src/shared/components/navbar.tsx

```diff
@@ -1,69 +1,126 @@
-import { Component, ChangeEvent, linkEvent } from "inferno";
+import { ChangeEvent, linkEvent } from "inferno";
 import { Link } from "inferno-router";
-import { LinkLine } from "./link-line";
-import { Icon } from "./icon";
-import { i18n, languages } from "../i18next";
+import { Icon, IconSize } from "./icon";
+import { i18n, LANGUAGES } from "../i18next";
+import classNames from "classnames";
+import { SELECT_CLASSES } from "./common";
 
-export class Navbar extends Component<any, any> {
-  constructor(props: any, context: any) {
-    super(props, context);
-  }
+const NavLink = ({ content }) => <li className="text-gray-400">{content}</li>;
 
-  handleLanguageChange(_: any, event: ChangeEvent<HTMLSelectElement>) {
-    location.href = `/?lang=${event.target.value}`;
-  }
+const NavLinks = () => (
+  <>
+    <NavLink
+      content={
+        <Link onClick={() => closeNavbarDropdown()} to="/instances">
```

Comment:
Since `closeNavbarDropdown` takes no args, you can just pass it in directly instead of calling it in a lambda function, i.e.:
```tsx
<Link onClick={closeNavbarDropdown} to="/instances">
```

You can do that with the other nav links as well.